xbmc/addons/binary/interfaces/api1/Addon/AddonCallbacksAddon.cpp

```diff
@@ -226,15 +227,19 @@ bool CAddonCallbacksAddon::GetAddonSetting(void *addonData, const char *strSetti
 
         if (id == strSettingName && !type.empty())
         {
-          if (type == "text"     || type == "ipaddress" ||
-              type == "folder"   || type == "action"    ||
-              type == "music"    || type == "pictures"  ||
-              type == "programs" || type == "fileenum"  ||
-              type == "file"     || type == "labelenum")
+          if (type == "text"       || type == "ipaddress" ||
```

Comment:
vertical alignment is crap. please refrain from doing so. there is always an expression that is longer than the longest one before. vertical alignment only ruins the diff.